packages/web3-eth-accounts/package.json

```diff
@@ -57,7 +57,8 @@
 		"typescript": "^4.7.4"
 	},
 	"dependencies": {
-		"@ethereumjs/tx": "^3.5.2",
+		"@ethereumjs/rlp": "^4.0.1",
```

Comment:
So, `@ethereumjs/rlp` is still used. Did you forget to push some code? Or, do I miss something?

The CHANGELOG.md contains:
> -   Moved @ethereumjs/tx, @ethereumjs/common, @ethereumjs/rlp code to our source code (#5963)

Thanks,

Reply:
Removed from changelog, thanks.
@jdevcs asked me to revert changes for RLP [here](https://github.com/web3/web3.js/pull/5963#discussion_r1159627500)